bgpd/bgp_fsm.c

```diff
@@ -562,42 +562,45 @@ void bgp_delayopen_timer(struct event *thread)
 }
 
 /* BGP Peer Down Cause */
-const char *const peer_down_str[] = {"",
-				     "Router ID changed",
-				     "Remote AS changed",
-				     "Local AS change",
-				     "Cluster ID changed",
-				     "Confederation identifier changed",
-				     "Confederation peer changed",
-				     "RR client config change",
-				     "RS client config change",
-				     "Update source change",
-				     "Address family activated",
-				     "Admin. shutdown",
-				     "User reset",
-				     "BGP Notification received",
-				     "BGP Notification send",
-				     "Peer closed the session",
-				     "Neighbor deleted",
-				     "Peer-group add member",
-				     "Peer-group delete member",
-				     "Capability changed",
-				     "Passive config change",
-				     "Multihop config change",
-				     "NSF peer closed the session",
-				     "Intf peering v6only config change",
-				     "BFD down received",
-				     "Interface down",
-				     "Neighbor address lost",
-				     "No path to specified Neighbor",
-				     "Waiting for Peer IPv6 LLA",
-				     "Waiting for VRF to be initialized",
-				     "No AFI/SAFI activated for peer",
-				     "AS Set config change",
-				     "Waiting for peer OPEN",
-				     "Reached received prefix count",
-				     "Socket Error",
-				     "Admin. shutdown (RTT)"};
+const char *const peer_down_str[] = {
+	"",
+	"Router ID changed",
+	"Remote AS changed",
+	"Local AS change",
+	"Cluster ID changed",
+	"Confederation identifier changed",
+	"Confederation peer changed",
+	"RR client config change",
+	"RS client config change",
+	"Update source change",
+	"Address family activated",
+	"Admin. shutdown",
+	"User reset",
+	"BGP Notification received",
+	"BGP Notification send",
+	"Peer closed the session",
+	"Neighbor deleted",
+	"Peer-group add member",
+	"Peer-group delete member",
+	"Capability changed",
+	"Passive config change",
+	"Multihop config change",
+	"NSF peer closed the session",
+	"Intf peering v6only config change",
+	"BFD down received",
+	"Interface down",
+	"Neighbor address lost",
+	"No path to specified Neighbor",
+	"Waiting for Peer IPv6 LLA",
+	"Waiting for VRF to be initialized",
+	"No AFI/SAFI activated for peer",
+	"AS Set config change",
+	"Waiting for peer OPEN",
+	"Reached received prefix count",
+	"Socket Error",
+	"Admin. shutdown (RTT)",
+	"Suppress Fib Turned on or Off",
```

Comment:
nit: Just to be consistent: `on or off` (lowercase, but not mixed).

Reply:
fixed